firefox-ios/firefox-ios-tests/Tests/XCUITests/BookmarksTests.swift

```diff
@@ -392,7 +383,7 @@ class BookmarksTests: BaseTestCase {
 
     private func bookmarkPageAndTapEdit() {
         bookmark()
-        app.buttons["Edit"].waitAndTap()
+        bookmark()
```

Comment:
do we need to go to bookmark() twice?

Reply:
This private function is for `testEditBookmark()`, a full functional test. This test bookmarks a page first and then opens the Bookmark page. Let me take a look what's going on here.

Reply:
Yes, we need to call `bookmark()` twice. Before the menu refactor, there's an "Edit" button from the toast to access the bookmarks panel right away.
![Simulator Screenshot - iPhone 15 Pro - 2024-11-21 at 16 41 44](https://github.com/user-attachments/assets/3688ed24-a0de-437c-8c23-017b10790c5b)

After the menu refactor, there's no such a button. The way to open the bookmarks panel is to open the browser tab menu ➡️ Save ➡️ Edit Bookmarks (was "Add Bookmarks"). The second `bookmarks()` call opens the bookmarks panel.
![Simulator Screenshot - iPhone 15 - 2024-11-21 at 17 20 39](https://github.com/user-attachments/assets/0861cac9-6550-4244-a575-8bb8a050adc8)